services/webhook/tests/test_precedent.py

```
@@ -0,0 +1,116 @@
+from __future__ import annotations
+
+from ledger import LedgerRow
+from personas.code_reviewer.precedent import (
+    ClassPrecision,
+    class_precision,
+    match_precedent,
+    render_precedent_note,
+)
+
+
+def _row(pr, cls, verdict, *, path="services/webhook/consumer.py", finding="x", ts=""):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_row`

(ANN202)